### DISCUSSION TITLE
nanoELS H1 BV20

### DISCUSSION THREAD

**Discussion by bestbean** [kachurovskiy/nanoels#109] [Show and tell]
Works nice!
Encoder 2000 steps, on 56t gear with 1/4 spindle rotation (in code 500 step encoder).
Stepper 70t gear on 84t gear lead screw (in code 5/6 lead screw mm).
Power supply and stepper driver on the backside of spindle housing.
![BV20-vorne](https://user-images.githubusercontent.com/13404673/220381281-81f0f6d1-b398-468d-b3e6-1f4c8c961e0c.JPG)
![nanoELS-BV20-1](https://user-images.githubusercontent.com/13404673/220381557-78c4e390-88e8-4654-b285-71c2a7673577.jpg)

**Comment by kachurovskiy** [comment]
Congrats on finishing the build! It's been more than a year 😃 These things do take time. https://github.com/kachurovskiy/nanoels/discussions/42

**Reply by bestbean** [reply]
Yes. I had already finished in March 2022, but the adapter stepper-to-gear was not strong enough for heavier use. [https://github.com/kachurovskiy/nanoels/discussions/56#discussioncomment-4760001](url)
